### PR TITLE
updated documentation and examples of grpc proxy

### PR DESCRIPTION
**Pull request by sredxny**
## Related Issue (if applicable)
None

## Description
When adding grpc services in Tyk is not required to set a custom port for each service, instead, the gateway can route correctly based in the service name. Hence, in this PR I update this information and simplify the examples 

## Motivation and Context
Give proper information, be consistent and simplify examples

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/key-concepts/grpc-proxy.md [hunk 1]
```diff
@@ -22,7 +21,8 @@ You can also perform [gRPC load balancing](#grpc-load-balancing).
 
 #### Prerequisites
 - Enable  HTTP/2 support on the Gateway side, for both incoming and upstream connections, by setting `http_server_options.enable_http2` and `proxy_enable_http2` to true in your `tyk.conf` Gateway config file.
-- Set `disable_ports_whitelist` to true, so the Gateway can use additional ports to expose the service.
+- Ensure that `strip listen path` is set to false in your Api.
```
Comment (letzya) [inline tyk-docs/content/key-concepts/grpc-proxy.md]: API

Comment (letzya) [reply tyk-docs/content/key-concepts/grpc-proxy.md]: Worth explaining the reasons for it

**Review by letzya — CHANGES_REQUESTED** [review]
Looks good 
Just 2 things
1. Please add some explanation based on your take away from the conversation that led to this pr.
2. Change Api to API throughout the pr

**Comment by sredxny** [comment]
made some changes

**Review by tbuchaillot — APPROVED** [review]
TIL 😄

**Review by mativm02 — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Pull request merged**